demos/browser/app/meetingV2/videofilter/SegmentationUtil.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { detect } from 'detect-browser';
+
+const SEGMENTATION_DEPENDENCIES = [
+  {
+    src: 'https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@2.7.0/dist/tf.min.js',
+    integrity: 'sha384-uI1PW0SEa/QzAUuRQ6Bz5teBONsa9D0ZbVxMcM8mu4IjJ5msHyM7RRtZtL8LnSf3',
+    crossOrigin: 'anonymous',
+  },
+
+  {
+    src: 'https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core@2.7.0/dist/tf-core.min.js',
+    integrity: 'sha384-DlI/SVdTGUBY5hi4h0p+nmC6V8i0FW5Nya/gYElz0L68HrSiXsBh+rqWcoZx3SXY',
+    crossOrigin: 'anonymous',
+  },
+
+  {
+    src:
+      'https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-webgl@2.7.0/dist/tf-backend-webgl.min.js',
+    integrity: 'sha384-21TV9Kpzn8SF68G1U6nYN3qPZnb97F06JuW4v0FDDBzW+CUwv8GcKMR+BjnE7Vmm',
+    crossOrigin: 'anonymous',
+  },
+
+  {
+    src: 'https://cdn.jsdelivr.net/npm/@tensorflow-models/body-pix@2.0.5/dist/body-pix.min.js',
+    integrity: 'sha384-dPJ/sICXCqdh39bsuGLVFcOiRyeL/XcFrwiFrJq9oh7k1TCtsUKhX6UV2X4UuKU4',
+    crossOrigin: 'anonymous',
+  },
+];
+
+
+export async function loadBodyPixDependency(timeoutMs: number): Promise<void> {
+  // the tf library loading order must be followed
+  for (let i = 0; i < SEGMENTATION_DEPENDENCIES.length; i++) {
+    const dep = SEGMENTATION_DEPENDENCIES[i];
+    await new Promise<void>((resolve, reject) => {
+      let script = document.createElement('script');
+      const timer = setTimeout(() => {
+        reject(new Error(`Loading script ${dep.src} takes longer than ${timeoutMs}`));
+      }, timeoutMs);
+      script.onload = function(_ev) {
+        clearTimeout(timer);
+        resolve();
+      };
+      script.src = dep.src;
+      script.integrity = dep.integrity;
+      script.crossOrigin = dep.crossOrigin;
```

Comment:
Set `src` last in case the browser aggressively prefetches before you set the integrity.